styles.css

```diff
@@ -0,0 +1,155 @@
+/*
+* Prefixed by https://autoprefixer.github.io
+* PostCSS: v8.4.14,
+* Autoprefixer: v10.4.7
+* Browsers: last 4 version
+*/
+
+body {
+    margin: 0 auto;
+    font-family: "Poppins", sans-serif;
+    background-image: url(./img/mobBg.png);
+}
+
+.mainMenu {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: justify;
+        -ms-flex-pack: justify;
+            justify-content: space-between;
+    background-color: #FFFFFF;
+}
+
+.menuLogo {
+    color: #6070FF;
+    font-weight: 700;
+    font-size: 24px;
+    padding-left: 4%;
+    line-height: 6%;
+}
+
+.menuIcon {
+    position: relative;
+    top: 12px;
+    width: 22px;
+    right: 24px;
+    
+}
+
+.menuLvl {
+    display: block;
+    background-color: #6070FF;
+    width: 100%;
+    height: 2.5px;
+    margin: 4px;
+    border-radius: 2px;
+}
+
+.homeContent {
+    padding-left: 8%;
+    padding-right: 8%;
+}
+
+.homeContent h1 {
+    font-weight: 700;
+    font-size: 3rem;
+    line-height: 64px;
+    padding-top: 20%;
+}
+
+.contentPara {
+    margin-top: -10%;
+    font-size: 1rem;
+}
+
+.contentLink {
+    margin-top: -3%;
+    font-size: 1rem;
+    font-weight: 500;
+    color: #7F8CFF;
+}
+
+.homeContent li {
+    list-style-type: none;
+    margin-left: -15%;
+}
+
+.homeContent img {
+    height: 20px;
+    width: 20px;
+    margin-right: 4%;
+}
+
+
+@media screen and (min-width: 375px) {
+    body {
+        background-image: url(./img/deskBg.png);
+        background-repeat: no-repeat;
+        background-size: cover;
+    }
+    
+    .homeContent {
+        padding-left: 8%;
+        padding-right: 8%;
+    }
+
+    .homeContent h1 {
+        font-size: 4rem;
+        line-height: 75px;
+        padding-top: 10%;
+    }
```

Comment:
- [ ] Compare your paragraph and the Figma one , I noticed that you forget to add the color used for this template kindly use the same color given for the design . and "glad to see" should be on the same line as we can see that on the design :wink: 
Your Design 
![Screenshot from 2023-02-17 16-26-53](https://user-images.githubusercontent.com/88894525/219682290-2fe869c7-0464-45a2-977b-cd802ba9b1e3.png)

Figma Design 
![Screenshot from 2023-02-17 16-27-14](https://user-images.githubusercontent.com/88894525/219682378-a4cf7cb0-4ec7-4f78-91e3-2e0985a9f4a5.png)

_I think you can use this `color: #172B4D`; to match the figma design_